charts/nm-vllm-grafana-dashboards/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+appVersion: "v0.0.1"
```

Comment:
I changed this to `v0.0.1` because there is no underlying app and the dashboards are the resource of interest and as such, might as well start versioning with `0.0.1`

Reply:
oops good call